File: HAI704I - Architectures Distribuées/TP2/TP2-Server/TP2-ServiceServer/model/agency/Agency.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TP2_Server.model.agency
{
    class Agency
    {
        public string Name
        {
            get;
        }

        private string login;
        private string password;

        public Agency(string name, string login, string password)
        {
            this.Name = name;
            this.login = login;
            this.password = password;
        }

        public bool Connect(string login, string password)
        {
            return this.login.Equals(login) && this.password.Equals(password);
        }
    }
}

File: HAI704I-ArchitecturesDistribuees/TP2/TP2-Java/TP2-Server/src/main/java/tp2/server/model/agency/Agency.java
package tp2.server.model.agency;

public class Agency
{
    private String name;

    private String login;
    private String password;

    public Agency(String name, String login, String password)
    {
        this.name = name;
        this.login = login;
        this.password = password;
    }

    public String getName()
    {
    	return this.name;
    }

    public boolean connect(String login, String password)
    {
        return this.login.equals(login) && this.password.equals(password);
    }
}
